transcom/mymove
# MAIN B-20451 - SC & TOO Creating Boat Shipment and Shipment Card Display

## [B-20451](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20451)

NOTE: This should get merged in after this Main PR: https://github.com/transcom/mymove/pull/13693

INT PR: 
https://github.com/transcom/mymove/pull/13619

REBASE INT PR: https://github.com/transcom/mymove/pull/13801

## Summary

This is to add ability for SC & TOO to create a Boat shipment in Move details page.
Also added a Boat shipment card to display shipment information in Move details page.

### How to test

1. Create any move with any shipment.
2. As a SC -> search for the move -> use `Add a new shipment` button and select `Boat` from the dropdown.
3. Fill out the required fields and click `Save` (NOTE: for dimensions - if requirements are not met, it should display a message in a red text).
4. You should see the boat shipment created with a shipment card displaying information you have entered.
5. Do the same for TOO and make sure it works.

## Screenshots
![image](https://github.com/user-attachments/assets/ece19275-9ceb-44b6-a367-3398cadc13a6)
![image](https://github.com/user-attachments/assets/74d18d62-0a35-40ae-85da-fe9cfd098bfa)
![image](https://github.com/user-attachments/assets/209c3720-e4a2-4795-b122-214ad7f28e5e)
